Sources/Widgets/Widgets3D/SeedWidget/behavior.js

```diff
@@ -0,0 +1,95 @@
+import macro from 'vtk.js/Sources/macros';
+
+export default function widgetBehavior(publicAPI, model) {
+  const state = model.widgetState;
+  const moveHandle = state.getMoveHandle();
+  moveHandle.setVisible(true);
+  // default color is red
+  moveHandle.setColor3(255, 0, 0);
+
+  model._isDragging = false;
+  model.previousPosition = null;
+  model.classHierarchy.push('SeedWidgetProp');
```

Comment:
For readability, I would recommend putting this line as the first line in widgetBehavior.

Reply:
Moved in [9c65bdd](https://github.com/Kitware/vtk-js/pull/2907/commits/9c65bddff30fba5a75bae17ddf292eeaf12ca86c)